karma.conf.js

```diff
@@ -1,128 +1,43 @@
         ],
 
         // list of files / patterns to load in the browser
         files: [
-            { pattern: 'node_modules/babel-polyfill/dist/polyfill.js', included: true, served: true },
-            { pattern: 'test/test.js', included: true, served: true }
+            
+            'test/index.js'
         ],
-
-        plugins: [
-            require('karma-webpack'),
-            require('karma-mocha'),
-            require('karma-phantomjs-launcher'),
-            require('karma-chrome-launcher'),
-            require('karma-safari-launcher'),
-            require('karma-firefox-launcher'),
-            require('karma-ie-launcher'),
-            require('karma-sinon-chai'),
-            require('karma-coverage'),
-            require('karma-spec-reporter'),
-            require('karma-sourcemap-loader')
-        ],
-
-        webpackMiddleware: {
-            noInfo: true,
-            stats: false
-        },
-
-        webpack: {
-            devtool: 'inline-source-map',
-
-            resolve: {
-                modules: [
-                    __dirname,
-                    'node_modules'
-                ]
-            },
-
-            module: {
-                rules: [
-                    {
-                        test: /\.js$/,
-                        exclude: /(dist|chai)/,
-                        loader: 'babel-loader',
-                        query: {
-                            presets: ['es2015'],
-                            plugins: [
-                                'transform-flow-strip-types',
-                                'transform-es3-property-literals',
-                                'transform-es3-member-expression-literals',
-                                ['transform-es2015-for-of', {loose: true}],
-                                ['flow-runtime', {
-                                    'assert': true,
-                                    'annotate': true
-                                }],
-                                [ '__coverage__', { only: `${__dirname}/src` } ]
-                            ]
-                        }
-                    },
-                    {
-                        test: /\.(html?|css|json)$/,
-                        loader: 'raw-loader'
-                    }
-                ]
-            },
-            bail: false,
-            plugins: [
-                new webpack.DefinePlugin({
-                    __TEST__: true
-                })
-            ]
+        preprocessors: {
+            'test/index.js': 'webpack'
         },
 
+        webpack: testConfig,
 
+        
         // list of files to exclude
         exclude: [
         ],
 
-
-        // preprocess matching files before serving them to the browser
-        // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
-        preprocessors: {
-            'test/test.js': ['webpack', 'sourcemap'],
-            'test/windows/**/*.js': ['webpack', 'sourcemap'],
```

Comment:
OK. Can live with that, so long as it's not just giving a line number in test.js. 👍